mdakit_membcurv/.gitignore

```diff
@@ -0,0 +1 @@
+*.npz
```

Comment:
I'd move the `.gitignore` to the top level of the repo.

Reply:
Ignore also
```
__pycache__
*.pyc
```

Reply:
Thanks @orbeckst. Is there a way I can double check the `__pycache__` got actually untracked? How can I identify that particular change? Not sure I can find it the "Files changed" tab.

Reply:
It's still there if you look under https://github.com/MDAnalysis/membrane-curvature/pull/9/files

If you do a 
```
git rm -rf  mdakit_membcurv/lib/__pycache__/
git commit -m 'removed __pycache__'
git push
```
then it should be removed from the PR.

It's important that you *Squash and merge* the PR because that will rewrite git history in such a way that it appears as if the deleted files were never there. Otherwise it's fairly difficult to truly get rid of files in git.